cabinets/__init__.py

```diff
@@ -59,14 +59,14 @@ def set_configuration(protocol, **kwargs):
     return cabinet_cls.set_configuration(**kwargs)
 
 
-def read(uri, raw=False, **kwargs):
+def read(uri, parser=True, **kwargs):
```

Comment:
`parser` probably is the right name here, it just feels like in the boolean cases `parse` would make sense. Though I do think it is very reasonable to interpret `parser=False` as, "don't use a parser," and `parser=True` as, "use a parser, but I'm not going to specify which one, so you choose."

Reply:
I prefer `parser` because you _can_ think of it as you described, and it obviously works well for custom classes, while `parse` works well for `True/False` but `parse=CustomParser` feels wrong.